src/taskManager/Task.java

```diff
@@ -0,0 +1,97 @@
+package taskManager;
+
+public class Task implements Taskable {
+    private String title;
+    private String description;
+    private String priority;
+
+    //TODO: use Enum for status
+    private String status;
+
+    public Task(String title, String description, String priority, String status) {
+        if (title.trim().equals("") ||
+                description.trim().equals("") ||
+                priority.trim().equals("") ||
+                status.trim().equals("")) {
+            System.out.println("Empty values are not allowed");
+            return;
+        }
```

Comment:
this won't help - task with empty fields will still be created :)

Reply:
got it :)